src/components/FeatureViews/AdminPortal/ActionLog.js

```diff
@@ -0,0 +1,482 @@
+                                        {objectOptions.map((objectType) =>
+                                            (<MenuItem key={objectType} value={objectType}>
+                                                {objectType}
+                                            </MenuItem>)
+                                        )}
+                                    </Select>
+                                </FormControl>
+                            </Grid>
+                        </TableCell>
+                        <TableCell>
+                            <Grid container>
+                                <Grid>
+                                    Details
+                                </Grid>
+                            </Grid>
+                            <Grid>
+                                <br />
+                            </Grid>
+                        </TableCell>
+                    </TableRow>
+                </TableHead>
+                <TableBody>
+                    {logData && logData.logs.results.map((actionItem) => {
+                        return (
+                            <TableRow>
+                                <TableCell>
+                                    <Moment
+                                        date={actionItem.date}
+                                        format="dddd, M/D/YYYY h:mm a"
+                                    />
+                                </TableCell>
+                                <TableCell>
+                                    {userOptions && userOptions.map((userType) => {
+                                        if (userType.user.id == actionItem.userId) {
+                                            return (`${userType.user.firstName} ${userType.user.lastName} #${actionItem.userId}`)
+                                        }
+                                    })}
+                                </TableCell>
+                                <TableCell>
+                                    <div style={{ textTransform: "capitalize" }}>
+                                        {actionItem.adminType.toLowerCase()}
+                                    </div>
+                                </TableCell>
+                                <TableCell>
+                                    {renderChip(actionItem.action)}
+                                </TableCell>
+                                <TableCell>
+                                    <div style={{ textTransform: "capitalize" }}>
+                                        {actionItem.objectType}
+                                    </div>
+                                </TableCell>
+                                <TableCell>
+                                    {actionItem.objectRepr}
+                                </TableCell>
+                            </TableRow>
+                        )
+                    })}
+                </TableBody>
+            </Table>
+            {logData &&
```

Comment:
same comment here - make sure you're using the `loading` variable to validate if `logData` is valid before coming to this return statement.